app/components/items/form.tsx

```diff
@@ -107,6 +107,7 @@ export const ItemForm = ({ title, category, description }: Props) => {
             label={zo.fields.description()}
             name={zo.fields.description()}
             defaultValue={description || ""}
+            placeholder="Add a description for your asset. Supports markdown."
```

Comment:
There is a type error because you removed the incoming code from dev branch in the `markdown-editor.tsx` file.